SPDX-FileCopyrightText: 2025 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: Artistic-1.0

mktables embeds a timestamp

diff --git perl-5.17.4/lib/unicore/mktables perl-5.17.4/lib/unicore/mktables
index 0f8268626c..a1b52d45c0 100644
--- perl-5.17.4/lib/unicore/mktables
+++ perl-5.17.4/lib/unicore/mktables
@@ -16167,7 +16167,7 @@ if ( $file_list and $make_list ) {
 #
 # $file_list -- File list for $0.
 #
-#   Autogenerated starting on $start_time ($localtime)
+#   Autogenerated
 #
 # - First section is input files
 #   ($0 itself is not listed but is automatically considered an input)
